samples/config_ptq_dfu2021_vgg11_without_preprocess_classification.yaml

```diff
@@ -0,0 +1,149 @@
+# affix version
+version:
```

Comment:
The config file looks quite heavy to understand since there are many parameters related to different phases of model development including training. Many parameters are commented here. I wonder if it is possible to make it shorter keeping only the calibration part and the relevant parameters.

Reply:
This is per UPenn's recommendation. This is an omni configuration file that contains training/testing and now calibration. We will discuss with UPenn team again on this design.